generator/lib/builder/om/PHP5ObjectBuilder.php

```diff
@@ -165,6 +165,9 @@ protected function getDefaultValueString(Column $col)
                 throw new EngineException(sprintf('Default Value "%s" is not among the enumerated values', $val));
             }
             $defaultValue = array_search($val, $valueSet);
+            if ($col->getPhpType() == 'string') {
+                $defaultValue = "'$defaultValue'";
```

Comment:
User manually sets ***sqlType*** which injects whatever there is into the generated query. Default comes from ```$valueSet``` which is supposed to be a valid CSV string inside ```ENUM(...)```.
I think it is possible to set a flawed SQL in ```sqlType``` and therefore put ```'``` into the default value but query will be broken before that and won't work anyway.
AFAIR xml schema won't validate with ```&quot;``` in the value so if there is a single quote, it will has to be escaped already.